packages/client/package.json

```diff
@@ -97,7 +97,7 @@
     "typescript": "5.2.2"
   },
   "engines": {
-    "node": "^18.15.0"
+    "node": ">=18"
```

Comment:
What about a range up to current stable (i.e. 20)?

Reply:
done